source/renderer/app/components/sidebar/Sidebar.js

```diff
@@ -32,20 +32,20 @@ type Props = {
 };
 
 export type SidebarMenus = {
-  wallets: ?{
-    items?: Array<SidebarWalletType>,
-    activeWalletId?: ?string,
-    actions?: {
+  wallets: {
+    items: Array<SidebarWalletType>,
+    activeWalletId: ?string,
+    actions: {
       onWalletItemClick: Function,
     },
-  },
-  hardwareWallets: ?{
-    items?: Array<SidebarHardwareWalletType>,
-    activeWalletId?: ?string,
-    actions?: {
+  } | null,
```

Comment:
@DeeJayElly instead `null` you can add `wallets: ?{` declaration. 
Same for `hardwareWallets`

Reply:
@DeeJayElly ignore my last comment. You did that in the last commit `440b035d6dd5db2a624c3e997725a24ce5cde1ae`